VimeoNetworking/Sources/VimeoRequestSerializer.swift

```diff
@@ -125,32 +125,32 @@ final public class VimeoRequestSerializer: AFJSONRequestSerializer
     
     // MARK: Private API
     
-    private func setup(apiVersion apiVersion: String)
+    private func setup(apiVersion: String)
     {
-        self.setValue("application/vnd.vimeo.*+json; version=\(apiVersion)", forHTTPHeaderField: self.dynamicType.AcceptHeaderKey)
+        self.setValue("application/vnd.vimeo.*+json; version=\(apiVersion)", forHTTPHeaderField: type(of: self).AcceptHeaderKey)
 //        self.writingOptions = .PrettyPrinted
     }
 
-    private func setAuthorizationHeader(request request: NSMutableURLRequest) -> NSMutableURLRequest
+    private func setAuthorizationHeader(request: NSMutableURLRequest) -> NSMutableURLRequest
```

Comment:
Search the code for `NSMutable` I think these should all just be `var URLRequest`